modules/core/src/v2/wallet.ts

```diff
@@ -484,6 +487,53 @@ export interface DownloadKeycardOptions {
   backupKeyID?: string;
 }
 
+// #region TSS interfaces
+
+export interface PrebuildTransactionWithIntentOptions {
+  reqId?: RequestTracer;
+  intentType?: string;
+  sequenceId?: string;
+  recipients?: {
+    address: string;
+    amount: string | number;
+  }[];
+  comment?: string;
+  memo?: Memo;
+  verification?: VerificationOptions;
+}
+
+export interface TSSSendManyOptions extends SendManyOptions {
+  reqId: RequestTracer;
+  intentType: string;
+}
+
+// find a better name
+export enum ShareKeyPosition {
```

Comment:
do we need to export this? looks like its only used in this file.
plz double check your other exports to make sure we're not over exposing info

Reply:
Removing every not needed export